### ISSUE TITLE
Colors not working

### ISSUE THREAD

**Issue by wdtracy** [jmihelcic/colored-regions#3]
Not sure if there is something wrong with my setup, but the color highlighting is not working for me:

![image](https://user-images.githubusercontent.com/632264/37117238-e641ef24-220d-11e8-8e7d-7bf200cf8a02.png)

The filename is test.js.  I also tried in a c# file and it didn't work.

Thanks,

Wade

**Comment by jmihelcic** [comment]
Hey!

I've tried it locally (put 0.2 instead of 0.5 just for image clarity, but tested with both), and it seems to work correctly. Did you try reloading VSCode or reinstalling the extension?

![test](https://user-images.githubusercontent.com/3227602/37117769-6cb4ea86-2252-11e8-9675-9dc74ba65703.PNG)

**Comment by wdtracy** [comment]
Yep, tried reloading, reinstalling, etc.  I wonder if I have another extension or setting somewhere that causes a conflict.

**Comment by wesleycouto** [comment]
Not working with me too

**Comment by VHeusdens** [comment]
same here unfortunately
